### PR TITLE
NOISSUE - Refine docs

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: drasko <drasko.draskovic@gmail.com>

### What does this do?
Breaks long `Getting Started` into separate dedicated chapters. Uses CLI for `Getting Started` example for simplicity.

### CODE REVIEW

**Review by manuio — DISMISSED** [review]
LGTM

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,609 +1,109 @@
...
 }
-```
 
-You can specify  `offset` and  `limit` parameters in order to fetch specific
-group of channels. In that case, your request should look like:
 
-```
-curl -s -S -i --cacert docker/ssl/certs/mainflux-server.crt --insecure -H "Authorization: <user_auth_token>" https://localhost/channels?offset=0&limit=5
-```
+"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1NDcwMjE3ODAsImlhdCI6MTU0Njk4NTc4MCwiaXNzIjoibWFpbmZsdXgiLCJzdWIiOiJmcmllbmRseV9iZWF2ZXJAZW1haWwuY29tIn0.Tyk31Ae680KqMrDqP895PRZg_GUytLE0IMIR_o3oO7o"
```
[the remark this answers is not in the document]
Comment (drasko) [reply docs/getting-started.md:45]: I am pasting the literal resoponse, so I will leve it like this for now...

**Review by nmarcetic — APPROVED** [review]

File: docs/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (anovakovic01) [reply docs/getting-started.md:45]: Yes, but you depend on token format, and every time it changes, you will have to update docs. IMHO, it's harder to maintain.

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Pull request merged**